### DISCUSSION TITLE
Do I need to  have a strip at all?

### DISCUSSION THREAD

**Discussion by gerroon** [tin2tin/Pallaidium#54] [Q&A] [answered]
When I generate from strips it complaints of no strip. Do I need to have vid strip? If so what kind of strip do I need to have in place?

**Comment by tin2tin** [comment]
What models do you have selected and what is your output? send me a screenshot of both add-on preferences and the sidebar options.

Ex. controlnet, open pose and img2img as video output needs strip input. The input strip is the strip selected when generating.

**Comment by gerroon** [comment]
This is what I was testing simply.

![image](https://github.com/tin2tin/Pallaidium/assets/8519469/52944c75-ac5c-448c-88f0-28250889513e)

![image](https://github.com/tin2tin/Pallaidium/assets/8519469/f7558125-7b82-46b4-b4a3-cbee1d96cefc)

**Comment by tin2tin** [comment, marked as answer]
In your 2. ss you have selected Input: Strips - which is why it complains if you haven't selected a strip. So change that to No Input, if you do not want to do ex. img2vid, vid2vid or text strip to video.
